Magento/magento_readme.md

```diff
@@ -0,0 +1,58 @@
+# Deploy Magento e-commerce on Azure
+

```

Comment:
Can we add a deploy to azure button .. this is what needs to be added 

[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-mysql%2Fmaster%2FMagento%2Fazuredeploy.json)

Reply:
Added to the new PR